### PR TITLE
Add new principle to the manifesto: "Start from the developer's experience"

### PR DESCRIPTION
**Pull request by pepibumur**
I'm replacing the *"We are not a project generator tool"* principle with a more generic and user-centric one, *"Start form the developer's experience"*.

In the years I've worked building tools both at Shopify and in my spare time, I noticed this is one of the biggest mistakes that leads us to build complex and not user-friendly solutions. I think Tuistfolks should start designing solutions from the problem and picturing how they'd like the user experience to be.

### CODE REVIEW

File: website/markdown/docs/contribution/manifesto.mdx [hunk 1]
```diff
@@ -45,14 +45,13 @@ Xcode is a great tool, but so many years of improvements, new platforms, and pro
 
 Tuist should take the opportunity to keep things simple because working on simple things is fun and motivates us. No one wants to spend time trying to debug an error that happens at the very end of the compilation process, or understanding why they are not able to run the app on their devices. Xcode delegates the tasks to its underlying build system and in some cases it does a very poor job translating errors into actionable items. Have you ever got a “framework X not found” error and you didn’t know what to do? Imagine if we got a list of potential root causes for the bug.
 
-## 5. We are not a project generation tool
+## 5. Start from the developers' experience
```
Comment (raven) [inline website/markdown/docs/contribution/manifesto.mdx]:
```suggestion
## 5. Start from the developer's experience
```

File: website/markdown/docs/contribution/manifesto.mdx [hunk 2]
```diff
@@ -45,14 +45,13 @@ Xcode is a great tool, but so many years of improvements, new platforms, and pro
...
 
-That means we sometimes have to say no to bringing ideas from Xcode that complicate scaling. For instance, if dealing with code signing at scale is something that Xcode doesn't do well, let's take the opportunity to re-think how code signing is done.
+We like to think as [Tom Preston](https://tom.preston-werner.com/) puts it in [this podcast](https://tom.preston-werner.com/): _"Most things can be achieved, whatever you have in your head you can probably pull of with code as long as is possible within the constrains of the universe"_. If we imagine **how we'd like the developer experience to be**, it's just a matter of time to pull it off — starting analyzing the problems from the developer experience gives us a unique point of view that will lead us to solutions that users will love to use.
```
Comment (raven) [inline website/markdown/docs/contribution/manifesto.mdx:52]: `you can probably pull of with code` -> `you can probably pull off with code`

`starting analyzing the` -> `by starting to analyze the`

```suggestion
We like to think as [Tom Preston](https://tom.preston-werner.com/) puts it in [this podcast](https://tom.preston-werner.com/): _"Most things can be achieved, whatever you have in your head you can probably pull off with code as long as is possible within the constrains of the universe"_. If we imagine **how we'd like the developer experience to be**, it's just a matter of time to pull it off — by starting to analyze the problems from the developer experience gives us a unique point of view that will lead us to solutions that users will love to use.
```

File: website/markdown/docs/contribution/manifesto.mdx [hunk 3]
```diff
@@ -45,14 +45,13 @@ Xcode is a great tool, but so many years of improvements, new platforms, and pro
...
+We might feel tempted to follow what everyone is doing, even if that means keeping on with the inconveniences that everyone continues to complain about. Let's not do that. _How do I imagine archiving my app? How would I love code signing to be? What processes can I help streamline with Tuist?_ For example, adding support for [Fastlane](https://fastlane.tools) is a solution to a problem that we need to understand first. We can get to the root problem or need by asking "why" questions. Once we narrow down where the motivation comes from, we can think of how Tuist can help them best. Maybe the solution is integrating with Fastlane, but it's important we don't disregard other equally valid solutions that we can put on the table before making trade-offs.
```
Comment (raven) [inline website/markdown/docs/contribution/manifesto.mdx]: `means keeping on with` -> `means sticking with`

```suggestion
We might feel tempted to follow what everyone is doing, even if that means sticking with the inconveniences that everyone continues to complain about. Let's not do that. _How do I imagine archiving my app? How would I love code signing to be? What processes can I help streamline with Tuist?_ For example, adding support for [Fastlane](https://fastlane.tools) is a solution to a problem that we need to understand first. We can get to the root of the problem by asking "why" questions. Once we narrow down where the motivation comes from, we can think of how Tuist can help them best. Maybe the solution is integrating with Fastlane, but it's important we don't disregard other equally valid solutions that we can put on the table before making trade-offs.
```

**Comment by pepibumur** [comment]
Addressed your suggestions @raven 👍

**Pull request merged**